### PR TITLE
[Fix] Pathlib replaced with os

### PR DESCRIPTION
**Pull request by YasheshSavani**
This PR will replace Pathlib based code to os based code.

### CODE REVIEW

**Review by manikd31 — APPROVED** [review]

**Comment by corneliusboehm** [comment]
Just out of curiosity: Why are we going back to `os.path`? I like `pathlib` :grin:

**Comment by manikd31** [comment]
> Just out of curiosity: Why are we going back to `os.path`? I like `pathlib` grin

@antoinemrcr suggested we shouldn't use `pathlib` and stick to `os.path` as we do in the other scripts as well. IMO, I also prefer `pathlib` :stuck_out_tongue:

**Comment by corneliusboehm** [comment]
So an alternative would have been to change the other scripts to `pathlib` as well, right? :grin: 
But if that has been decided already, I'll stay quiet. Most of all, I'm for consistency as well :wink:

**Comment by antoinemrcr** [comment]
I don t like pathlib, i prefer os

**Comment by corneliusboehm** [comment]
Should we call a vote? :P

**Review by antoinemrcr — DISMISSED** [review]
LGTM

**Comment by sunny-panchal** [comment]
> Should we call a vote? :P

I don't have a strong opinion...for consistency we could stick to os for now, but in the long run, why do you prefer os over pathlib @antoinemrcr? And why pathlib over os @corneliusboehm?

File: scripts/tools/flip_video.py [hunk 1]
```diff
@@ -17,29 +17,30 @@
...
 from docopt import docopt
-from pathlib import Path
 
 if __name__ == '__main__':
+    # Parse arguments
     args = docopt(__doc__)
-    videos_path_in = Path.cwd() / Path(args['--path_in'])
-    videos_path_out = Path.cwd() / Path(args['--path_out']) if args.get('--path_out') else videos_path_in
+    videos_path_in = os.path.join(os.getcwd(), args['--path_in'])
+    videos_path_out = os.path.join(os.getcwd(), args['--path_out']) if args.get('--path_out') else videos_path_in
```
Comment (sunny-panchal) [inline scripts/tools/flip_video.py:28]:
```suggestion
    videos_path_in = os.path.join(os.getcwd(), args['--path_in'])
    videos_path_out = os.path.join(os.getcwd(), args.get('--path_out', args['--path_in']))
```

**Comment by antoinemrcr** [comment]
pathlib is not straight forward to understand, there is a learning curve that make the project harder to understand

File: scripts/tools/flip_video.py (hunk 1 above)
Comment (YasheshSavani) [reply scripts/tools/flip_video.py]: This change is giving NoneType error

**Review by manikd31 — APPROVED** [review]
Looks good to me :+1:

**Review by sunny-panchal — APPROVED** [review]
works :+1:

**Pull request merged**